conjure_python_client/_lib/types.py

```diff
@@ -36,6 +37,14 @@ def __init__(self, item_type):
         self.item_type = item_type
 
 
+class SetType(ConjureType):
+    item_type = None  # type: Type[DecodableType]
```

Comment:
It looks like we're pretty inconsistent elsewhere, but fields should probably be underscored with a `@property` method for access.